lib/Resource.js

```diff
@@ -158,7 +158,7 @@ Resource.prototype.get = function (resource, finished, query, cb) {
         result = utils.makeEmptyResult('resource',
                                        code,
                                        'The resource couldn\'t be retrieved');
-      if (error) {
```

Comment:
I don't understand the new condition. error || !error is always true, right? What's the condition to tell apart that an error occurred or not

Reply:
Sorry, that was a mistake -- the condition reads (error || !response), handling the case where response may be undefined. I made the fix in my local PS install, then brought it manually to the bindings repos.